src/main/java/com/brightpath/learnify/controller/mapper/DtoMapper.java

```diff
@@ -58,6 +59,7 @@ public NoteSummaryDto asNoteSummaryDto(Note note) {
                 .createdAt(note.createdAt())
                 .type(asNoteTypeDto(note.type()))
                 .pagesCount(note.pagesCount())
+                .accessType(toResourceAccessTypeDto(note.permissionLevel()))
```

Comment:
we do not want optionals?

Reply:
I don't think Optional would be useful here. We expect nullable value in controller and JPA query